.github/workflows/create-test-lint-wf-template.yml

```diff
@@ -66,6 +68,8 @@ jobs:
         exclude:
           - TEMPLATE: github
           - TEMPLATE: is_nfcore
+          - TEMPLATE: nf_core_configs
+            profile: "self_hosted_runner"
```

Comment:
is the indentation correct here?

Reply:
I think so :) we are removing the map with these values, and we do the same [some lines above](https://github.com/nf-core/tools/blob/master/.github/workflows/create-test-lint-wf-template.yml#L49-L51)